salt/modules/swarm.py

```diff
@@ -0,0 +1,440 @@
+# -*- coding: utf-8 -*-
+"""
+Docker Swarm Module using Docker's Python SDK
+=============================================
+
+:codeauthor: Tyler Jones <jonestyler806@gmail.com>
+
+.. versionadded:: Oxygen
+
+The Docker Swarm Module is used to manage and create Docker Swarms.
+
+Dependencies
+------------
+
+- Docker installed on the host
+- Docker python sdk >= 2.5.1
+
+Docker Python SDK
+-----------------
+pip install -U docker
+
+More information: https://docker-py.readthedocs.io/en/stable/
+"""
+# Import python libraries
+from __future__ import absolute_import
+import salt.config
+import salt.loader
+import json
+
+try:
+    import docker
+    HAS_DOCKER = True
+except ImportError:
+    HAS_DOCKER = False
+
+__virtualname__ = 'docker'
```

Comment:
this line will overwrite the docker module, you will want to return `swarm` or just not return True from the `__virtual__` function, since the file is already named swarm.py